Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/event/model/Document.java

```diff
@@ -36,7 +34,8 @@
  * @author Naresh Subramanyan
  */
 public class Document {
-
+    
+    private final String SIZE_IN_KILO_BYTES = "KB";
```

Comment:
Just a suggestion: SIZE_IN_KILO_BYTES makes me thing the name of this constant is a number (a size).  Maybe use something like "KILOBYTE_LABEL" or "KILOBYTE_DISPLAY"